### PR TITLE
Add changelog entries for the 22.08 release

### PR DESCRIPTION
**Pull request by the-mikedavis**
Changelog notes for the next release 🚀

Update: time is a bit short in July so we'll shoot to release in August as 22.08

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,193 @@
+# 22.07 (2022-07-29)
+
+A big _thank you_ to our contributors! This release had 78 contributors.
+
+As usual, the following is a summary of each of the changes since the last release.
+For the full log, check out the [git log](https://github.com/helix-editor/helix/compare/22.03..22.05).
```
Comment (univerz) [inline CHANGELOG.md:6]: 22.05..22.07

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,198 @@
...
+- Add clipboard information to logs and the healthcheck ([#3271](https://github.com/helix-editor/helix/pull/3271))
+- Fix align selection behavior on tabs ([#3276](https://github.com/helix-editor/helix/pull/3276))
+
+Themes:
+
+- Update `tokyonight` and `tokyonight_storm` themes ([#2606](https://github.com/helix-editor/helix/pull/2606))
+- Update `solarized_light` themes ([#2626](https://github.com/helix-editor/helix/pull/2626))
+- Fix `catpuccin` `ui.popup` theme ([#2644](https://github.com/helix-editor/helix/pull/2644))
+- Update selection style of `night_owl` ([#2668](https://github.com/helix-editor/helix/pull/2668))
+- Fix spelling of `catppuccin` theme ([#2713](https://github.com/helix-editor/helix/pull/2713))
```
Comment (sudormrfbin) [inline CHANGELOG.md:106]: Wouldn't this be a breaking change?

Comment (the-mikedavis) [reply CHANGELOG.md:106]: Yeah good call. We should probably try to get https://github.com/helix-editor/helix/pull/3281 into this release too so we don't get a reputation for breaking catppuccin every release 😛

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,217 @@
...
+- OpenSCAD ([#2680](https://github.com/helix-editor/helix/pull/2680))
+- Prisma ([#2703](https://github.com/helix-editor/helix/pull/2703))
+- Clojure ([#2780](https://github.com/helix-editor/helix/pull/2780))
+- Starlark ([#2903](https://github.com/helix-editor/helix/pull/2903))
+- Elvish ([#2948](https://github.com/helix-editor/helix/pull/2948))
+- Fortran ([#3025](https://github.com/helix-editor/helix/pull/3025))
+- Ungrammar ([#3048](https://github.com/helix-editor/helix/pull/3048))
+- SCSS ([#3074](https://github.com/helix-editor/helix/pull/3074))
+- Go Template ([#3091](https://github.com/helix-editor/helix/pull/3091))
+- Graphiz dot ([#3241](https://github.com/helix-editor/helix/pull/3241))
```
Comment (DrSensor) [inline CHANGELOG.md:168]: Typo?

**Review by archseer — APPROVED** [review]

**Pull request merged**